src/components/skeletons/RegisterPageSkeleton/RegisterPageSkeleton.tsx

```diff
@@ -0,0 +1,49 @@
+import Skeleton from 'react-loading-skeleton';
+
+import Container from 'components/ui/Container';
+
+import 'react-loading-skeleton/dist/skeleton.css';
+import styles from './RegisterPageSkeleton.module.css';
+
+const RegisterPageSkeleton = () => {
+  return (
+    <Container className={styles.wrapper}>
+      <div className={styles.registerForm}>
+        <Skeleton className={styles.title} />
+        <div className={styles.formInputBlock}>
+          {' '}
+          <ul className={styles.inputs}>
+            <li>
+              <Skeleton className={styles.input} />
+            </li>
+            <li>
+              <Skeleton className={styles.input} />
+            </li>
+            <li>
+              <Skeleton className={styles.input} />
+            </li>
+            <li>
+              <Skeleton className={styles.input} />
+            </li>
+          </ul>
+          <Skeleton className={styles.photoBlock} />
+        </div>
+        <Skeleton className={styles.button} />
+        <Skeleton className={styles.additional} />
+      </div>
+      <ul className={styles.stepsBlock}>
+        <li>
+          <Skeleton className={styles.stepCardOne} />
+        </li>
+        <li>
+          <Skeleton className={styles.stepCardTwo} />
+        </li>
+        <li>
+          <Skeleton className={styles.stepCardThree} />
+        </li>
+      </ul>
```

Comment:
map

Reply:
it is easier to style each card in this way as they all are different.